node/src/chain_spec.rs

```diff
@@ -178,7 +178,7 @@ pub fn finney_mainnet_config() -> Result<ChainSpec, String> {
 		Some("bittensor"),
 		None,
 		// Properties
-		None,
+		Some(properties),
```

Comment:
Does changing this still make a difference? I thought `finnet_mainnet_config` is really only relevant for the genesis block

Reply:
Technically yes, but the properties value in the chain-spec isn't considered apart of genesis so it can be changed on the nodes without having to restart chain history.